src/site/content/en/blog/media-recording-audio/index.md

```diff
@@ -130,32 +130,45 @@ Web Audio API is a simple API that takes input sources and connects those
 sources to nodes which can process the audio data (adjust Gain etc.) and
 ultimately to a speaker so that the user can hear it.
 
-One of the nodes that you can connect is a `ScriptProcessorNode`. This node will
-emit an `onaudioprocess` event every time the audio buffer is filled and you
-need to process it. At this point, you could save the data into your own buffer
-for later use.
+One of the nodes that you can connect is an `AudioWorkletNode`. This node gives
+you the low-level capability for custom audio processing. The actual audio
+processing happens in the `process()` callback method in the `AudioWorkletProcessor`.
+Call this function to feed inputs and parameters and fetch outputs.
+
+Checkout [Enter Audio Worklet](https://developer.chrome.com/blog/audio-worklet/)
```

Comment:
I think this be "Check out" instead.

Reply:
Fixed this! Are you happy to sign off on the little bit of JS? @mjwilson-google

Reply:
Thanks for the fix!  I didn't check the JavaScript in detail; if you still need a full review please ping me again (may not get to it today).